recipes/jpy/build.sh

```diff
@@ -0,0 +1,19 @@
+set -ex
+
+export PBR_VERSION=$PKG_VERSION
+
+cd $SRC_DIR/jpy
+# Use PEP 517 to generate the wheel
+$PYTHON setup.py --maven bdist_wheel
+cd dist/
+ls -l 
+
+# Install wheel manually
+# $PYTHON -m pip install *.whl
+
+mkdir -p $PREFIX/jpy_wheel
+echo $PY_VER > /tmp/p_ver.txt
+
+VER=$( echo $PY_VER | tr -d '.' )
+
+cp -v $( find $SRC_DIR -name "jpy*$VER*$VER*.whl" ) $PREFIX/jpy_wheel 
```

Comment:
This is a very convoluted way to create a conda-package from a wheel. It seems that a direct `pip install`, as recommended in our docs, would work here, no?